pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -669,6 +698,33 @@
   csd_page_faq_question_06: "How does CS Discoveries fit with the other Code.org courses?"
   csd_page_faq_answer_06_01: "CS Discoveries is designed to fit naturally between our [CS Fundamentals](%{csf_url}) courses and our [CS Principles](%{csp_url}) course. While each of these courses is designed to be an age-appropriate entry point to computer science, students with previous experience will find many new topics to explore, and they will revisit familiar topics in novel and more challenging contexts."
 
+  csc_page_top_desc: "Make connections between computer science (CS) and other subjects like math, language arts, science, and social studies."
+  csc_page_intro_heading: "Teach CS while covering core subject standards"
+  csc_page_intro_list_01: "Incorporate CS into other subjects"
+  csc_page_intro_list_02: "Reinforce learning in other subjects"
+  csc_page_intro_list_03: "Meet core subject standards"
+  csc_page_intro_list_04: "Fun way to teach students about computer science"
+  csc_page_intro_list_05: "Open-ended projects for more personalized learning"
+  csc_page_intro_list_06: "Easy for any teacher with any background"
+  csc_page_curriculum_glance_topics: "Interdisciplinary"
+  csc_page_curriculum_glance_devices: "Laptop, Chromebook, Tablet"
+  csc_page_curriculum_glance_tools: "Sprite Lab"
+  csc_page_curriculum_glance_professional_learning: "Self-paced Modules"
+  csc_page_curriculum_glance_professional_languages: "English"
+  csc_page_curriculum_glance_professional_accessibility_features: "Text-to-speech, Closed captioning, Immersive reader"
+  csc_page_quote: "I greatly appreciate how thorough and well paced the lessons are. I am not familiar with computer science, but with the lessons you provide I feel confident in helping my students grasp the concepts and tie them into our curriculum."
+  csc_page_find_a_module_desc: "Our current CS Connections offerings are for 3rd to 5th grade classrooms and are designed to take 1 week of 45-minute lessons each day."
+  csc_page_faq_question_01: "Is CS Connections mapped to standards?"
+  csc_page_faq_answer_01: "Yes! CS Connections are mapped to the Common Core State Standards (CCSS) for ELA and Math, the Next Generation Science Standards (NGSS), and the Computer Science Teacher Association Standards (CSTA). "
```

Comment:
Nit: the question refers to CS Connections as singular with "Is" but the answer refers to it as a plural with "Are". I'm not sure which makes more sense but it would probably be good to have both be consistent